models/analysis/revenuecat_monthly_recurring_revenue.sql

```diff
@@ -44,8 +50,13 @@ final as (
 
     left join
         transaction_monthly_revenue
-        on date_spine.date_month >= transaction_monthly_revenue.start_time::date
-        and last_day(date_spine.date_month, month) <= last_day(transaction_monthly_revenue.effective_end_time::date, month)
+        on transaction_monthly_revenue.start_time::date <= last_day(date_month, month)
+        and case 
+                when transaction_monthly_revenue.effective_end_time >= convert_timezone('America/Los_Angeles', 'UTC', SYSDATE()::date) -- Assumes that RevenueCat runs their data model at this time. This gives an 99.9% accurate MRR number for the current month & prior months
```

Comment:
sysdate()::date vs current_date?

Reply:
sysdate is always in UTC

Reply:
Then why are you converting it haha?!

Reply:
If sysdate is UTC, you are converting it as if its in America time - so the timestamp will be out by 8 hours lol

Reply:
That's right I believe RevenueCat runs their data model at midnight Pacific Standard Time time since their HQ is located there. This gave me an accuracy of 99.99% in MRR if I move the timestamp by just a few hours the accuracy of the current month MRR drops immediately to <99%